ocs_ci/helpers/helpers.py

```diff
@@ -4776,3 +4776,22 @@ def is_rbd_default_storage_class(custom_sc=None):
 
     logger.error("Storageclass {default_rbd_sc} is not a default  RBD StorageClass.")
     return False
+def custom_round(number):
+    """Rounds a number down for values ending in exactly ".5".
+
+    Args:
+      number: The number to round.
+
+    Returns:
+      The rounded number (down for values ending in ".5").
+    """
+    # Convert the number to a string to check the decimal part
+    number_str = str(number)
+
+    # Check if the decimal part exists and ends in ".5"
+    if "." in number_str and number_str.endswith(".5"):
```

Comment:
Hey. 
Func may cause the bugs

1. Input: 2.504
Expected Output: 3 (since it's slightly more than 2.5)
Actual Output: 2 (since .endswith(".5") check fails due to floating-point representation)

2. Negative numbers and numbers such as 3.49 will also cause the bug. 

We don't need this function in helpers, it is not universal and to specific to your needs, please move the logic to the test or function to the test module.

Reply:
Hello Daniel, this function was specifically return for cli values and as per my testing I have never seen cli giving out values more than single decimal value, and the values wont be negative either as this are storage utilisation values. After carefully looking over all the values that are obtained by the cli I have added this round function. As this is not a common function I will move it out from the helpers file and add it into blockpool file.

Reply:
done